homeassistant/components/panasonic_viera/remote.py

```diff
@@ -39,11 +41,12 @@ async def async_setup_entry(
     async_add_entities([PanasonicVieraRemoteEntity(remote, name, device_info)])
 
 
-class PanasonicVieraRemoteEntity(RemoteEntity):
+class PanasonicVieraRemoteEntity(PanasonicVieraEntity, RemoteEntity):
```

Comment:
Why is the available property not overridden for this class in the same way as for media player?